<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Person Ex Machina</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/images/computerman.png</normaloff>:/icons/images/computerman.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tableTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="personsTableTab">
       <attribute name="title">
        <string>Persons</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Operations</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="personAddButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="personEditButton">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="personDeleteButton">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="personFilterCheckBox">
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="personSearchComboBox">
                <property name="currentText">
                 <string/>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="personSearchField"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="personTable">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="personResultsLabel">
          <property name="text">
           <string>Results: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="machinesTableTab">
       <attribute name="title">
        <string>Machines</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Operations</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="machinesAddButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="machinesEditButton">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="machinesDeleteButton">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QCheckBox" name="machinesFilterCheckBox">
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="machinesSearchComboBox">
                <property name="currentText">
                 <string/>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="machinesSearchField"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="machineTable"/>
        </item>
        <item>
         <widget class="QLabel" name="machinesResultsLabel">
          <property name="text">
           <string>Results: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="connectionTableTab">
       <attribute name="title">
        <string>Connections</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Operations</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="addConnectionButton">
                <property name="text">
                 <string>Add connection</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="connectionsDeleteButton">
                <property name="text">
                 <string>Delete connection</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QCheckBox" name="connectionsFilterCheckBox">
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="connectionsSearchComboBox">
                <property name="currentText">
                 <string/>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="connectionsSearchField"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="connectionsTable"/>
        </item>
        <item>
         <widget class="QLabel" name="connectionsResultsLabel">
          <property name="text">
           <string>Results: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="typesTab">
       <attribute name="title">
        <string>Types</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Operations</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QPushButton" name="typeAddButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="typeEditButton">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="typeDeleteButton">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_8">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QCheckBox" name="typeFilterCheckBox">
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="typesSearchComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="currentText">
                 <string/>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="typeSearchField"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="typesTable"/>
        </item>
        <item>
         <widget class="QLabel" name="typesResultLabel">
          <property name="text">
           <string>Results: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="systemsTab">
       <attribute name="title">
        <string>Systems</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QGroupBox" name="groupBox_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Operations</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QPushButton" name="systemAddButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="systemEditButton">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="systemDeleteButton">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_10">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QCheckBox" name="systemFilterCheckBox">
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="systemSearchComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="currentText">
                 <string/>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="systemSearchField"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="systemsTable"/>
        </item>
        <item>
         <widget class="QLabel" name="systemResultLabel">
          <property name="text">
           <string>Results: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDeletePerson">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/cross.png</normaloff>:/icons/images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionEditPerson">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/minus.png</normaloff>:/icons/images/minus.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionPersonConnectToMachine">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/right.png</normaloff>:/icons/images/right.png</iconset>
   </property>
   <property name="text">
    <string>Connect to machine...</string>
   </property>
  </action>
  <action name="actionDeleteMachine">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/cross.png</normaloff>:/icons/images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionEditMachine">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/minus.png</normaloff>:/icons/images/minus.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionConnectToPerson">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/right.png</normaloff>:/icons/images/right.png</iconset>
   </property>
   <property name="text">
    <string>Connect to person...</string>
   </property>
  </action>
  <action name="actionDeleteConnection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/cross.png</normaloff>:/icons/images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete connection</string>
   </property>
  </action>
  <action name="actionDeleteType">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/cross.png</normaloff>:/icons/images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionEditType">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/minus.png</normaloff>:/icons/images/minus.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDeleteSystem">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/cross.png</normaloff>:/icons/images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionEditSystem">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/minus.png</normaloff>:/icons/images/minus.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
